.github/ISSUE_TEMPLATE/browser.md

```diff
@@ -1,23 +1,28 @@
 ---
-name: Bug
-about: 'Bug - please file against relevant browser instead'
+name: Browser/implementation bug
+about: 'Bug? Please file against relevant browser/implementation instead'
```

Comment:
Not sure that "Bug?" is doing much for us here. Feels like it should either be removed and implied by the name or expanded to something like "For bugs in a WebGPU implementation, please..."